From 717f994be2466d378e6611a4739c606db6d0dc46 Mon Sep 17 00:00:00 2001
From: Wenzong Fan <wenzong.fan@windriver.com>
Date: Sun, 25 Oct 2015 22:44:47 -0400
Subject: [PATCH] autogen.sh: fix find-version for beta checking

find-version always assumes that gnupg is beta if autogen.sh is run
out of git-repo. This doesn't work for users whom just take release
tarball and re-run autoconf in their local build dir.

Upstream-Status: Pending

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
---
 autogen.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index 7effd56..d673432 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -228,7 +228,6 @@ if [ "$myhost" = "find-version" ]; then
       rvd=$((0x$(echo ${rev} | head -c 4)))
     else
       ingit=no
-      beta=yes
       tmp="-unknown"
       rev="0000000"
       rvd="0"
-- 
1.9.1

